.circleci/config.yml

```diff
@@ -121,39 +121,6 @@ jobs:
           name: "Deploy new docs"
           command: ./.circleci/deploy-docs.sh
 
-  docs-python35:
```

Comment:
Let's move to a 3.7 docker build, and pin numpy 1.11 on the 3.6 build?
The CircleCI build does complement the coverage of the test suite.